src/components/Coins/Coins.css

```diff
@@ -0,0 +1,69 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #5788e4;
+  max-width: 100%;
+  box-sizing: border-box;
+}
+
+.search input {
+  display: block;
+  margin: 0 auto;
+  margin-top: 20px;
+  padding: 10px;
+  background-color: #5788e4;
+  border: 2px solid #fff;
+  color: #fff;
+}
+
+input::placeholder {
+  color: #fff;
+}
+
+input:focus {
+  border: 2px solid #fff;
+  outline: none;
+}
+
+.coins {
+  columns: 2;
+  gap: 0;
+  width: 100%;
+  box-sizing: border-box;
+  margin-top: 20px;
+}
+
+.coin-card {
+  width: 100%;
+  min-height: 200px;
+  background-color: #5384e1;
+  padding: 25px 0;
+  box-sizing: border-box;
+}
+
+.odd-bg {
+  background-color: #4c7bd3;
+}
```

Comment:
- [ ] Please make the light and dark background of the coin cards staggered like a chessboard, not like straps as you did to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/197370306-bbd441df-a28d-40a5-81f7-71e76c6b95bc.png)  | ![image](https://user-images.githubusercontent.com/68796232/197370313-86e70630-6c86-45e7-a189-9f9d172932ff.png)  |